lighthouse-core/audits/installable-manifest.js

```diff
@@ -89,17 +89,20 @@ class InstallableManifest extends MultiCheckAudit {
   /**
    * @param {LH.Artifacts} artifacts
    * @param {LH.Audit.Context} context
-   * @return {Promise<{failures: Array<string>, manifestValues: LH.Artifacts.ManifestValues}>}
+   * @return {Promise<{failures: Array<string>, manifestValues: LH.Artifacts.ManifestValues, manifestUrl: string | undefined}>}
    */
   static async audit_(artifacts, context) {
     const manifestValues = await ManifestValues.request(artifacts, context);
     const manifestFailures = InstallableManifest.assessManifest(manifestValues);
 
+    const manifestUrl = artifacts.WebAppManifest ? artifacts.WebAppManifest.manifestUrl : undefined;
+
     return {
```

Comment:
side note how do we have an audit that doesn't have a `score`.. so confused

Reply:
`extends MultiCheckAudit` and that `audit_` life, def confusing though

Reply:
OHHHH MultiCheckAudit. of course.